dialect/sql/schema/sqlite.go

```diff
@@ -148,5 +198,143 @@ func (d *SQLite) dropIndex(ctx context.Context, tx dialect.Tx, idx *Index, table
 }
 
 // fkExist returns always true to disable foreign-keys creation after the table was created.
-func (d *SQLite) fkExist(context.Context, dialect.Tx, string) (bool, error) { return true, nil }
-func (d *SQLite) table(context.Context, dialect.Tx, string) (*Table, error) { return nil, nil }
+func (d *SQLite) fkExist(ctx context.Context, tx dialect.Tx, table *Table, fk *ForeignKey) (bool, error) {
+	rows := &sql.Rows{}
+
+	if err := tx.Query(ctx, fmt.Sprintf("pragma foreign_key_list('%s')", table.Name), []interface{}{}, rows); err != nil {
+		return false, fmt.Errorf("sqlite3: reading table foreign keys description %v", err)
+	}
+	defer rows.Close()
+
+	for rows.Next() {
+		var (
+			id        int
+			seq       int
+			tableName string
+			from      string
+			to        string
+			onUpdate  string
+			onDelete  string
+			match     string
+		)
+		if err := rows.Scan(&id, &seq, &tableName, &from, &to, &onUpdate, &onDelete, &match); err != nil {
+			return false, errors.Wrap(err, "while querying foreign keys")
+		}
+
+		if tableName == fk.RefTable.Name && fk.Columns[0].Name == from && fk.RefColumns[0].Name == to {
+			return true, rows.Close()
```

Comment:
Why not:
```suggestion
			return true, nil
```

You already call `defer rows.Close()` above.